platform/ios/src/MGLMapView.mm

```diff
@@ -3324,12 +3324,10 @@ - (void)_setVisibleCoordinates:(const CLLocationCoordinate2D *)coordinates count
     {
         latLngs.push_back({coordinates[i].latitude, coordinates[i].longitude});
     }
+    
+    CLLocationDirection cameraDirection = direction >= 0 ? direction : 0;
```

Comment:
What you have now is probably more readable, but something like `fmaxf(direction, 0)` would also work.